docs/content/end-users/configuration.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Create a Porter Config File
+description: How to customize Porter through configuration file
+aliases:
+- operators/configuration/
+---
+
+Porter's default behavior, such as log level, default plugins, etc., can be modified through its config file.
+The config file also provides the ability to define plugin specific
+configurations, so you don't need to repeat them each time when you run porter.
+
+## Location
+
+The configuration file should be stored in the PORTER_HOME directory, by default
+~/.porter/. In the example below, we are going to use the yaml format, however,
+porter supports other file format for its config file as well. For more detailed
+information, please see [configuration](/configuration/#config-file).
+
+First check if you have PORTER_HOME set on your machine:
+```console
+$ echo $PORTER_HOME
+```
+
+If it returns nothing, that means your PORTER_HOME is the default value,
+~/.porter/. For consistency of this example, let's create the environment
+variable like so:
+```console
+$ echo PORTER_HOME=~/.porter/
```

Comment:
Shouldn't this do not use the `echo` command for the environment variable to be defined and then stored?